src/open_inwoner/js/components/search/filter-mobile.js

```diff
@@ -0,0 +1,35 @@
+export class FilterMobile {
+  static selector = '.filter__mobile--toggle'
+
+  constructor(node) {
+    this.node = node
+    this.filterList = node.closest('.filter') // Find the closest parent with class 'filter'
```

Comment:
`.closest()` is often a kludge.

In general component hydration should work from the outside in: the selector selects the outer element, and the constructor grabs sub-elements chained deeper in that node. 

The idea of components is they're self contained, so when using closest() you're just grabbing and attaching to random parts outside the components responsibility. Sometimes it is necessary when there is a shared thing outside the component but is it here?